src/Application/Persistence/RetrySafeEntityManager.php

```diff
@@ -167,14 +167,18 @@ public function refresh($object, ?int $lockMode = null): void
     public function flush($entity = null): void
     {
         try {
+            /** @psalm-suppress TooManyArguments - passing non-null is depreacted, and
+             * \Doctrine\Persistence\ObjectManager::flush is declared with no args,
+             * but \Doctrine\ORM\EntityManager::flush still takes the arg for now.
+             */
             $this->entityManager->flush($entity);
         } catch (EntityManagerClosed $closedException) {
             $this->logger->error(
                 'EM closed. RetrySafeEntityManager::flush() trying with a new instance,' .
                 $closedException->__tostring()
             );
             $this->resetManager();
-            $this->entityManager->flush($entity);
+            $this->entityManager->flush();
```

Comment:
```suggestion
            $this->entityManager->flush($entity);
```
Finishes the revert for the same reasons as line 174